pkg/lib/config/feature_oauth.go

```diff
@@ -39,16 +39,18 @@ var _ = FeatureConfigSchema.Add("OAuthClientFeatureConfig", `
 		"maximum": { "type": "integer" },
 		"soft_maximum": { "type": "integer" },
 		"custom_ui_enabled": { "type": "boolean" },
-		"app2app_enabled": { "type": "boolean" }
+		"app2app_enabled": { "type": "boolean" },
+		"logo_uri_in_authn_page_enabled": { "type": "boolean" }
 	}
 }
 `)
 
 type OAuthClientFeatureConfig struct {
-	Maximum         *int  `json:"maximum,omitempty"`
-	SoftMaximum     *int  `json:"soft_maximum,omitempty"`
-	CustomUIEnabled *bool `json:"custom_ui_enabled,omitempty"`
-	App2AppEnabled  *bool `json:"app2app_enabled,omitempty"`
+	Maximum                   *int  `json:"maximum,omitempty"`
+	SoftMaximum               *int  `json:"soft_maximum,omitempty"`
+	CustomUIEnabled           *bool `json:"custom_ui_enabled,omitempty"`
+	App2AppEnabled            *bool `json:"app2app_enabled,omitempty"`
+	LogoURIInAuthnPageEnabled *bool `json:"logo_uri_in_authn_page_enabled,omitempty"`
 }
```

Comment:
I thought this should be a portal ui feature flag but not a flag in core feature config?